config/initializers/locale.rb

```diff
@@ -0,0 +1,4 @@
+# Whitelist locales available for the application
+I18n.available_locales = %i[en ja]
```

Comment:
Does this need any sort of translation file to not 💥? Should we hold off adding `ja` until the translation is added?